### PR TITLE
Extract default help header of console commands into a separate method.

### PR DESCRIPTION
**Pull request by ww-stas**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | no

I would like to overite  default help header but copy of all source code of getDefaultHelp method is wrong way.

### CODE REVIEW

File: framework/console/controllers/HelpController.php [hunk 1]
```diff
@@ -437,4 +437,13 @@ protected function getScriptName()
     {
         return basename(Yii::$app->request->scriptFile);
     }
+
+    /**
+     * Display a default help header.
+     * @return string default help header.
```
Comment (samdark) [inline framework/console/controllers/HelpController.php]: Need `@since 2.0.11` here.

File: framework/console/controllers/HelpController.php [hunk 2]
```diff
@@ -437,4 +437,13 @@ protected function getScriptName()
     {
         return basename(Yii::$app->request->scriptFile);
     }
+
+    /**
+     * Display a default help header.
```
Comment (samdark) [inline framework/console/controllers/HelpController.php]: It's not displaying, it's returning.

**Review by samdark — CHANGES_REQUESTED** [review]
Makes sense overall but fixes I've mentioned should be done. Additionally we need a CHANGELOG update.

File: framework/console/controllers/HelpController.php (hunk 1 above)
Comment (diezztsk) [inline framework/console/controllers/HelpController.php]: Fixed.

**Review by dynasource — APPROVED** [review]

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**